src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/JournalførVedtaksbrev.kt

```diff
@@ -148,35 +164,56 @@ class JournalførVedtaksbrev(
             vedlegg = vedlegg,
             behandlingId = vedtak.behandling.id,
             avsenderMottaker = utledAvsenderMottaker(mottakerInfo),
-            tilVerge = mottakerInfo.erVerge
+            tilVerge = tilManuellMottakerEllerVerge
```

Comment:
```suggestion
            tilManuellMottakerEllerVerge = tilManuellMottakerEllerVerge
```
Kanskje innafor å rename parameter navnet på journalførDokument metoden gitt at det nå brukes til manuellmottaker også